elements/pfe-readtime/README.md

```diff
@@ -0,0 +1,71 @@
+# PatternFly Element | Readtime element
+It takes in the word count of a given section and does a calculation on that number to return an estimated read time.
+
+## Usage
+
+```html
+    <!-- Default slot -->
+    <pfe-readtime for="readtime1">-minute readtime</pfe-readtime>
+```
+
+### Accessibility
+Explain how this component meets accessibility standards.
+
+## Slots
+
+- `namedSlot`: There is only one slot and it contains a <span> with the readtime value.
+
+## Attributes
+
+- `attr`: word-count is the data-attribute you will leverage for readtime. Example: word-count="2500" will let the component know that there is 2500 words in that section and it will do it's calculations based on that number. If you don't want to/have a data-attribute to leverage you can use id="readtime1" and it will get the word count for you.
+
+## Variable hooks
+
+Available hooks for styling:
+
+| Variable name | Default value | Region |
+| --- | --- | --- |
+| `--pfe-pfe-readtime--Color` | `#252527` | N/A |
```

Comment:
You can add:

```
:root { @include pfe-debug-log; }
```

in your sass file after the $LOCAL-VARIABLES map and copy the output in the _temp/pfe-readtime.css.  Be sure to delete that line before committing though.